ansible/roles/dokuwiki/files/initialize.d/dokuwiki

```diff
@@ -92,4 +272,6 @@ render(data / 'conf/acl.auth.php')
 render(data / 'conf/local.php', title="wiki")
 render(data / 'conf/plugins.local.php')
 render(data / 'conf/users.auth.php')
+(data / 'conf/dokuwiki.php').unlink()
```

Comment:
Maybe `render` should always overwrite its target files?

Reply:
nope, just that one file, because the other files have random private keys in there that we don't wanna overwrite

Reply:
fine.